pages/common/hub.md

```diff
@@ -31,3 +27,7 @@
 - Upload the current (local-only) repository to your GitHub account:
 
 `hub create`
+
+- Fetch git objects from upstream and update local branches:
```

Comment:
```suggestion
- Fetch Git objects from upstream and update local branches:
```

Reply:
OK. We may open a hub PR! (since I copied/pasted it 😉)

Reply:
Thank you  @marchersimon. Done in `fixup` commit 2edb1eef.

Reply:
> OK. We may open a `hub` PR! (since I copied/pasted it) 😉

let's go for it :) .. if there are any discrepancies there